### DISCUSSION TITLE
SSH Channels

### DISCUSSION THREAD

**Discussion by pukozs75** [iptv-org/community#324] [Show and tell]
I found a site ssh101 com It has a lot of balkan and other sports and entertainment channels.
If the channels that are contained in this site are not DMCA'd it would be really nice to add them in the playlist.

**Comment by BellezaEmporium** [comment]
I've heard of SSH101 before. It's maintained by Tulix Systems, the ones behind Giniko, BozzTV, etc etc...

I believe we used to have them before but they've changed their links.

**Comment by Sphinxroot** [comment]
easy to grab  no referrer  neeeded anymore to work  https://ssh101.bozztv.com/ssh101/xxxxxxx/playlist.m3u8
